pkgs/applications/misc/libation/default.nix

```diff
@@ -0,0 +1,107 @@
+{ lib
+, buildDotnetModule
+, fetchFromGitHub
+, dotnetCorePackages
+
+, copyDesktopItems
+, makeDesktopItem
+, wrapGAppsHook
+
+, libX11
+, libICE
+, libSM
+, libXi
+, libXcursor
+, libXext
+, libXrandr
+, fontconfig
+, glew
+
+, gtk3
+, gdk-pixbuf
+}:
+
+buildDotnetModule rec {
+  pname = "libation";
+  version = "11.0.3";
+
+  src = fetchFromGitHub {
+    owner = "rmcrackan";
+    repo = "Libation";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-PNsGIQ2h3r9y/m+3s9pmSQSXm6h5Z2GEvsgXSyeGvFs=";
+  };
+  sourceRoot = "source/Source";
+
+  dotnet-sdk = dotnetCorePackages.sdk_7_0;
+  dotnet-runtime = dotnetCorePackages.runtime_7_0;
+
+  nugetDeps = ./deps.nix;
+
+  dotnetFlags = [ "-p:PublishReadyToRun=false" ];
+
+  projectFile = [
+    "LibationAvalonia/LibationAvalonia.csproj"
+    "LibationCli/LibationCli.csproj"
+    "HangoverAvalonia/HangoverAvalonia.csproj"
+  ];
+
+  executables = [ "Libation" "LibationCli" "Hangover" ];
+
+  nativeBuildInputs = [ wrapGAppsHook copyDesktopItems ];
+
+  runtimeDeps = [
+    # For Avalonia UI
+    libX11
+    libICE
+    libSM
+    libXi
+    libXcursor
+    libXext
+    libXrandr
+    fontconfig
+    glew
+
+    # For file dialog
+    gtk3
+  ];
+
+  desktopItems = [
+    (makeDesktopItem {
+      name = pname;
+      icon = pname;
+      exec = meta.mainProgram;
+      desktopName = "Libation";
+      comment = "Liberate your Audiobooks";
+      categories = [ "Audio" ];
+      terminal = false;
+    })
+  ];
+
+  postInstall = ''
+    install -Dm644 ${src}/Images/libation_glass.svg \
+        $out/share/icons/hicolor/scalable/apps/${pname}.svg
+  '';
+
+  dontWrapGApps = true;
+
+  preFixup = ''
+    makeWrapperArgs+=("''${gappsWrapperArgs[@]}")
+  '';
+
+  postFixup = ''
+    cd $out/bin
+    mv Libation libation
+    mv LibationCli libationcli
+    mv Hangover hangover
```

Comment:
```suggestion
    mv $out/bin/Libation $out/bin/libation
    mv $out/bin/LibationCli $out/bin/libationcli
    mv $out/bin/Hangover $out/bin/hangover
```

Reply:
Seems the aarch64-darwin builder is running on a case-insensitive file system, the fix is perhaps to use an intermediate filename. If you do use that solution, note the reason why in a comment

Reply:
It now overrides the default dotnet fixup hook